frameworks/AutoGluon/exec.py

```diff
@@ -0,0 +1,114 @@
+import logging
```

Comment:
Regarding CatBoost files polluting the working directory, this has been fixed and will be included in the upcoming 0.0.7 release.

PR with fix: https://github.com/awslabs/autogluon/pull/446